src/main/scripts/run.py

```diff
@@ -53,11 +49,14 @@ def log_response(response):
 
 if __name__ == '__main__':
     # Parse args.
-    parser = argparse.ArgumentParser(description='Start the application server.')
+    parser = argparse.ArgumentParser(description='Starts the application server.')
     parser.add_argument('-d', '--debug', action='store_true')
     args = vars(parser.parse_args())
-    # Initialise logger.
-    init_logger(debug=args['debug'])
+    # Initialise blueprints
+    app.register_blueprint(verify)
+    app.register_blueprint(extract)
     # Run the server.
+    hutts_logger.disable_flask_logging(app)
     print('* Running server')
```

Comment:
I know you didn't touch this line, but would it be able for you to make it use `logger.info` for this statement instead of `print`? It should specify that the server is starting and on which port.